src/core/lib/security/authorization/matchers.cc

```diff
@@ -19,6 +19,7 @@
 #include <grpc/grpc_security_constants.h>
 
 #include "src/core/lib/address_utils/sockaddr_utils.h"
+#include "src/core/lib/security/security_connector/tls/tls_security_connector.h"
```

Comment:
I think this include has implications for the work that @yihuazhang is doing to remove insecure builds.  I think this would mean that the authz policy would depend on the TLS creds implementation, which means it would drag in a dependency on the SSL libraries.  That seems maybe more heavyweight than what we want here.

Can we maybe define the transport security type in some other file, so that it doesn't drag in the SSL dependency?

Reply:
Sounds good. Moved the type to grpc_security_constants file. PTAL.